build/cocos2d_libs.xcodeproj/project.pbxproj

```diff
@@ -17490,21 +17490,21 @@
 				HEADER_SEARCH_PATHS = "";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
 				LIBRARY_SEARCH_PATHS = (
-					"\"$(SRCROOT)/../external/png/prebuilt/ios\"",
```

Comment:
To be honest i don't want to include this change. Since we can not test tvos in China. And we haven't maintained it for a long time. Ricardo add this support, and he just tested on simulator only. I just want to make it as it is now.